Sources/Private/Model/Extensions/Bundle.swift

```diff
@@ -19,3 +20,13 @@ extension Bundle {
     #endif
   }
 }
+
+extension String {
+  fileprivate func removingJSONSuffix() -> String {
+    guard hasSuffix(".json") else {
```

Comment:
A comment explaining why we specifically only do this for `.json` files would be nice